src/components/select/SelectItemCommons.tsx

```diff
@@ -42,13 +42,30 @@ function withSelectItemWrapper<P extends SelectOption>(
   optionType: SelectOptionType,
 ) {
   const Wrapper = forwardRef<HTMLDivElement, P>((props, ref) => {
-    const { helpText, label, prefix, suffix, value, ...rest } = props;
+    const { helpText, label, prefix, selected, suffix, value, ...mantineAndCustomProps } = props;
+
+    const {
+      'aria-selected': _ariaSelected,
+      className,
+      'data-disabled': _dataDisabled,
+      'data-hovered': _dataHovered,
+      'data-selected': _dataSelected,
+      onMouseDown,
+      onMouseEnter,
+      tabIndex,
+      ...customProps
+    } = mantineAndCustomProps;
+
     return (
-      <SelectItemWrapper ref={ref} {...rest}>
-        <SelectItemStatus optionType={optionType} {...props} />
+      <SelectItemWrapper ref={ref} selected={selected} {...mantineAndCustomProps}>
```

Comment:
`SelectItemWrapper` will receive more than it needs but it's no big deal, it was already the case. It's easier than explicitly extract the Mantine props.

Reply:
That's fine, we should not think too much about it, that's the way mantine works if you don't pass a custom component anyway, I think it's good enough if we just remove our own stuff (helpText, prefix, suffix).

Also let's have the `selected` part of `mantineAndCustomProps`